client/sniffer/src/test/resources/create.bash

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
```

Comment:
nice! maybe rename the file to create_nodes_info_output ?

Reply:
Sure!